app/source.js

```diff
@@ -534,6 +535,21 @@ window.Source = function(templates, cwd, tm, source, revlayers, examples, isMapb
 
         this.model.save(attr, options);
         return ev && !! $(ev.currentTarget).is('a');
+
+		// Track layer count
+  		analytics.track('final upload layer count', { layercount: attr.layercount });
+
+		// Track each layers buffer size
+		analytics.track('buffer layer', { buffer: attr.layername });
+		analytics.track('buffer size setting', { bufferlayer: attr.layer.buffer-size });
+
+		// Track max and min zooms
+        analytics.track('maxzoom setting', { maxzoom: attr.maxzoom });
+        analytics.track('minzoom setting', { minzoom: attr.minzoom });
+
+		// Track file size on save
+		analytics.track('file size', { filesize: filesize });
+		
```

Comment:
want to consistify the indenting here? Studio is using 4 spaces in many places for tabs.